chandra_aca/tests/test_maude_decom.py

```diff
@@ -369,3 +369,46 @@ def test_start_stop():
     with pytest.raises(ValueError, match='Maximum allowed'):
         start, stop = (686111020, 686219020)
         _ = maude_decom.get_aca_packets(start, stop)
+
+
+def test_dynbgd_decom():
+    # This test looks at telemetry data around times when either BGDTYP or PIXTLM change.
```

Comment:
Nice test description.  In general these should be in a docstring not comment.  That way it is in the test function `__doc__` and can be introspected / documented.

Reply:
I just tossed this comment into the docstring.

Reply:
And that change caused unrelated flake8 fails so I'm guessing the flake8 changed or the file hadn't been checked in previous versions.